test-infra/prowjobs/unittests/main.sh

```diff
@@ -44,8 +44,26 @@ for TARGET in "${TARGETS[@]}"; do
   fi
 
   # Output test report.
+  cat test.out
   cat test.out | go-junit-report > ${ARTIFACTS}/${TARGET//\//_}_report.xml
   rm test.out
 done
 
+# Execute all unittests.sh scripts in their directories.
+# Each script should APPEND coverage data to GOCOVPATH or PYCOVPATH.
+RET=0
+find . -type f -name "unittests.sh" | while read script; do
+  # Change to the containing directory and run script.
+  cd $(dirname $script)
+  ./$(basename $script)
+
+  UNITTEST_RET=$?
+  if [ $RET == 0 ]; then
+    RET=$UNITTEST_RET
+  fi
+
+  popd
```

Comment:
`popd` pairs with `pushd`, not `cd`

Reply:
yuck, this was cruft from when I was trying to identify the current test suite code ([guest-test-infra](https://github.com/GoogleCloudPlatform/guest-test-infra) is where I landed).